web/modules/weather_data/src/Service/AlertUtility.php

```diff
@@ -638,4 +638,34 @@ public static function getPlacesFromAlertDescription($alert)
         // so return false so we don't do any other processing elsewhere.
         return false;
     }
+
+    /* Determine the "level" of the alert (warning, watch, advisory, etc..)
+     * based on the the last word in the alert event name
+     * */
+    public static function getAlertLevel($event)
+    {
+        $eventWords = explode(" ", $event);
+        $alertLevel = strtolower($eventWords[array_key_last($eventWords)]);
+        return $alertLevel;
+    }
```

Comment:
I think this is *mostly* fine, but there's an alert type for `evacuation immediate` and it's pretty high-priority so I'm guessing we don't want to classify it as an `other`. There's also `civil emergency message`, and both are prioritized above hurricane warning.

Let's roll with this for now, but I'm going to create a debt ticket here. I think it might make sense to have a different data structure for our list of known events and the high-level level can be associated directly to the event. But that doesn't need to be dealt with here.

Reply:
Noted in #1407